### PR TITLE
aptos_governance -> supra_governance

### PR DESCRIPTION
**Pull request by sjoshisupra**
Rename `aptos_governance` to `supra_governance`

### CODE REVIEW

File: aptos-move/framework/supra-framework/sources/aptos_governance.spec.move [hunk 1]
```diff
@@ -1,4 +1,4 @@
-spec supra_framework::aptos_governance {
```
Comment (ghost) [inline aptos-move/framework/supra-framework/sources/aptos_governance.spec.move:1]: @sjoshisupra We should also rename the file.

File: aptos-move/framework/supra-framework/doc/aptos_governance.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
-<a id="0x1_aptos_governance"></a>
+<a id="0x1_supra_governance"></a>
 
-# Module `0x1::aptos_governance`
+# Module `0x1::supra_governance`
```
Comment (ghost) [inline aptos-move/framework/supra-framework/doc/aptos_governance.md:4]: We should also rename the file.

**Review by ghost — CHANGES_REQUESTED** [review]
One thing I notice for the json file at `supra/aptos-core/api/goldens/aptos_api__tests__transactions_test__test_get_transactions_param_limit_exceeds_limit.json` also uses `aptos_governance`. Should we also rename that?

File: aptos-move/framework/supra-framework/doc/aptos_governance.md (hunk 2 above)
Comment (sjoshisupra) [reply aptos-move/framework/supra-framework/doc/aptos_governance.md:4]: We can ignore this one, doc is auto-generated and it creates `supra_governance.md` automatically.

File: aptos-move/framework/supra-framework/sources/aptos_governance.spec.move (hunk 1 above)
Comment (sjoshisupra) [reply aptos-move/framework/supra-framework/sources/aptos_governance.spec.move:1]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**